integrations/java/flink/src/main/java/com/hopsworks/tutorials/flink/fraud/SimProducer.java

```diff
@@ -61,14 +61,21 @@ public static void main(String[] args) throws Exception {
       .required(true)
       .hasArg()
       .build());
+  
+    options.addOption(Option.builder("parallelism")
+      .argName("parallelism")
+      .required(true)
```

Comment:
Can you check that the example arguments we pass in the example in the README also includes parallelism since we now require it?

Reply:
This one https://github.com/logicalclocks/hopsworks-tutorials/blob/master/integrations/java/flink/README.md